### PR TITLE
changed date format

### PR DESCRIPTION
**Pull request by Biki-das**
Changed the date format from dd/mm/yy to MM/YY, which is the standard for input of Expiry date of debitCards

### CODE REVIEW

**Comment by teoli2003** [comment]
@Rumyra Can you help here? (and/or maybe you can give the rights to commit here too?)

**Comment by Biki-das** [comment]
> @Rumyra Can you help here? (and/or maybe you can give the rights to commit here too?)

ok , thanks for the feedback

File: html/forms/html-form-structure/payment-form.html [hunk 1]
```diff
@@ -81,9 +81,9 @@ <h2>Payment information</h2>
               <label for="date">
                 <span>Expiration date:</span>
                 <strong><abbr title="required">*</abbr></strong>
-                <em>formatted as mm/dd/yyyy</em>
+                <em>formatted as MM/YY</em>
               </label>
-              <input type="date" id="date" name="expiration">
+              <input type="text" id="date" name="expiration" placeholder="MM/YY">
```
Comment (Rumyra) [inline html/forms/html-form-structure/payment-form.html:86]:
```suggestion
              <input type="text" id="date" name="expiration">
```

We don't need the placeholder here, as the format is stated in the label. Also it should reflect the example on the corresponding mdn page 👍

**Review by Rumyra — CHANGES_REQUESTED** [review]

File: html/forms/html-form-structure/payment-form.html (hunk 1 above)
Comment (Biki-das) [reply html/forms/html-form-structure/payment-form.html:86]: Ok, but the format is wrong for card expiry date format, if we don't use the placeholder then how should i approach the problem, i can make the format mm/yy, using JavaScript

**Comment by Biki-das** [comment]
Thank you for the suggestion @Rumyra  i have commited with the chahges you requested.

**Comment by hamishwillee** [comment]
FWIW I might do a bit more than this. Now its a text form the user can put anything they like in. You can but in a max length, min length, placeholder, and ideally a pattern.

**Comment by Biki-das** [comment]
> FWIW I might do a bit more than this. Now its a text form the user can put anything they like in. You can but in a max length, min length, placeholder, and ideally a pattern.

Thanks

**Comment by Biki-das** [comment]
I have tried to implement the expiry date format , please review @Rumyra  @hamishwillee .
PR link-https://github.com/mdn/learning-area/pull/397

File: html/forms/html-form-structure/payment-form.html [hunk 2]
```diff
@@ -81,9 +81,9 @@ <h2>Payment information</h2>
               <label for="date">
                 <span>Expiration date:</span>
                 <strong><abbr title="required">*</abbr></strong>
-                <em>formatted as mm/dd/yyyy</em>
+                <em>formatted as dd/mm/yyyy</em>
```
[the remark this answers is not in the document]
Comment (Biki-das) [reply html/forms/html-form-structure/payment-form.html:84]: pulled a PR and added the card expiry date format @hamishwillee @Rumyra please review

**Comment by Biki-das** [comment]
hello @teoli2003  can you ping anyone regarding this PR #397 , haven't got any response , hope for a feedback.

**Comment by hamishwillee** [comment]
@Biki-das Thanks. We're all pretty busy - I'll try review the PR today.

**Pull request closed without merging**